napari_clusters_plotter/_clustering.py

```diff
@@ -575,7 +575,7 @@ def result_of_clustering(returned):
 @catch_NaNs
 def mean_shift(
     reg_props: pd.DataFrame, quantile: float = 0.2, n_samples: int = 50
-) -> (str, np.ndarray):
+) -> tuple[str, np.ndarray]:
```

Comment:
I had it also this type annotation once like this, but it gives TypeError in Github Cl, so you need to revert it back how it was 😅

Reply:
Hey both,

you can use Tuples as return type annotation, *but* you have to import it specifically!

To use tuples for returns, you'd have to:

```Python
from typing import Tuple

def function(...) -> Tuple(int, float):
```

Reply:
Ahh thanks, I learned something new!

Reply:
thanks so much @jo-mueller! That seems like the better solution.

Reply:
You're welcome. Just as a side note: You can also returns lists like this with 

```
from typing import List

def function() -> List[]:
```

I learned about that just the other day [here](https://stackoverflow.com/questions/40181344/how-to-annotate-types-of-multiple-return-values) - but it does not work with Napari  ^^" If you wanted to return multiple layers, you'd always have to use `->List[LayerDataTuple]`